examples/eval-tests.dx

```diff
@@ -542,10 +542,10 @@ def cmpEither (x:Int|Int) (y:Int|Int) : Bool = x == y
 
 -- TODO: within-module version of this (currently fails in Imp checking)
 upperBound = isum $ for i:(Fin 4). 1
-for j:(Fin upperBound). 1.0
+:p for j:(Fin upperBound). 1.0
 > [1.0, 1.0, 1.0, 1.0]
 
-(for i:(Fin upperBound). 1, for j:(Fin 2). 2)
+:p (for i:(Fin upperBound). 1, for j:(Fin 2). 2)
```

Comment:
Not right now, but I'm often wondering if we should forbid naked expressions at the top level (it might allow us to give better parse errors, for example).